datalad/plugin/tests/test_addurls.py

```diff
@@ -245,22 +248,20 @@ def json_stream(data):
     return stream
 
 
-@known_failure_githubci_win
 def test_extract():
     info, subpaths = au.extract(
         json_stream(ST_DATA["rows"]), "json",
         url_format="{name}_{debut_season}.com",
         filename_format="{age_group}//{now_dead}//{name}.csv")
-
     eq_(subpaths,
-        ["adult", "kid", "adult/no", "adult/yes", "kid/no"])
+        [str(Path(p)) for p in ["adult", "kid", "adult/no", "adult/yes", "kid/no"]])
```

Comment:
A better way would be:

```
["adult", "kid", os.path.join("adult", "no"), os.path.join("adult", "yes"), os.path.join("kid", "no")]
```

or, if you want the line to be as compressed as possible:

```
[os.path.join(*parts) for parts in [["adult"], ["kid"], ["adult", "no"], ["adult", "yes"], ["kid", "no"]]]
```

Reply:
Either way is fine.

Reply:
makes sense, thanks!